histomicstk/PodoSighter_cnn_folder/Complete_CNN_Prediction.py

```diff
@@ -66,17 +76,39 @@
 print(resol)
 print(watershed_dist_thre)
 print(args.jsonout)
-print(args.outxml1)
+print(out_xml)
 print(tissue_thickness)
 print(csv_file_name)
+print(girder_api_url)
+print(girder_token)
+
 
+'''Set Girder API and Token'''
+'''============================='''
+
+folder = args.inputfolder
+girder_folder_id = folder.split('/')[-2]
+_ = os.system("printf 'Using data from girder_client Folder: {}\n'".format(folder))
+file_name = args.inputsvs.split('/')[-1]
+print(file_name)
+gc = girder_client.GirderClient(apiUrl=girder_api_url)
+gc.setToken(girder_token)
+files = list(gc.listItem(girder_folder_id))
+# dict to link filename to gc id
+item_dict = dict()
+for file in files:
+    d = {file['name']:file['_id']}
+    item_dict.update(d)
+itemID = item_dict[file_name]
+print(item_dict)
+print(itemID)
 
 try:
-    if species_name =='human':
+    if species_name =='Human':

```

Comment:
Where do we pass in `species_name` here? I would also recommend to use a `enum` class here. See [this](https://docs.python.org/3/library/enum.html) documentation. You can define the class on top of this file as (feel free to change the names):

```
from enum import Enum
class SpeciesName(Enum):
    Human = 'Human'
    Rat = 'Rat'
    Mouse = 'Mouse'
```

And then you can check each type as:
`species_name == SpeciesName.Human.value`

Reply:
switched to enum class in latest commit